.github/workflows/checks.yml

```diff
@@ -88,3 +88,51 @@ jobs:
         run: npm run typecheck
       - name: Run tests
         run: npm run test:unit
+
+  E2E:
+    # Only run e2e tests, if Build job is successful
+    needs: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - name: Install deps
+        run: npm install -g npm@8 && npm ci
+      - name: Start app in background
+        run: npm run serve &
+      - name: Checkout Playwright app
+        uses: actions/checkout@v3
+        with:
+          repository: balancer-labs/frontend-e2e
+          # Checkout a specific branch
+          ref: feature/use-playwright
```

Comment:
Remove this line when branch `feature/use-playwright` is merged to `balancer-labs/frontend-e2e` master